software/samba/en.md

```diff
@@ -33,22 +33,28 @@ sudo systemctl status nmb smb
 # Stop Samba manually
 sudo systemctl stop nmb smb
 
-# Configure Samba to not start automatically on each boot
-sudo systemctl disable nmb smb
+# Configure Samba to not start automatically on each boot and stop it now
```

Comment:
IMO `stop it now` should be changed to `immediately stop the service`. It's less concise but slightly clearer.